src/app/shared/log-in/ldap/dynamic-ldap.component.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * /users/sign-in
+ * @class LogInComponent
+ */
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'ds-dynamic-ldap',
+  templateUrl: './dynamic-ldap.component.html',
+  styleUrls: ['./dynamic-ldap.component.scss'],
+
+})
+export class DynamicLdapComponent {
```

Comment:
I would not implement this component in this PR

Reply:
Deleted the LDAP component.